modules/tools/migration/src/main/java/org/wso2/mb/migration/DBConnector.java

```diff
@@ -535,6 +596,66 @@ private boolean queueNameHasCapitals(String queueName) {
     private boolean bindingKeyHasCapitals(String bindingKey) {
         return !bindingKey.equals(bindingKey.toLowerCase());
     }
+
+    /**
+     * Close the prepared statement resource.
+     *
+     * @param preparedStatement PreparedStatement
+     */
+    protected void close(PreparedStatement preparedStatement) {
+        try {
+            if (preparedStatement != null && !preparedStatement.isClosed()) {
+                preparedStatement.close();
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * close the connection, prepared statement and the result set.
+     *
+     * @param connection        connection to be closed
+     * @param preparedStatement PreparedStatement
+     * @param resultSet         result set to be closed
+     */
+    private void close(Connection connection, PreparedStatement preparedStatement, ResultSet resultSet) {
+        close(resultSet);
+        close(preparedStatement);
+        close(connection);
+    }
+
+    /**
+     * closes the result set resources
```

Comment:
Doc comment should start with a capital letter and end with a period. Let's correct in all places